### PR TITLE
Deprecated kubectl flag "create-external-load-balancer"

### LINKED ISSUE #13070: Get rid of last remnants of "create-external-load-balancer"
There's still a kubectl flag at least, maybe more.  would be nice to make it deprecated in v1.1 if it isn't already, or nuke it if it was already deprecated.

### PR DESCRIPTION
**Pull request by janetkuo**
`create-external-load-balancer` ties to the old implementation of load-balancers and is totally obsoleted by `--type` for `kubectl expose`.  

Fixes #13070.

### CODE REVIEW

File: pkg/kubectl/cmd/expose.go [hunk 1]
```diff
@@ -63,8 +63,10 @@ func NewCmdExposeService(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().String("protocol", "TCP", "The network protocol for the service to be created. Default is 'tcp'.")
 	cmd.Flags().Int("port", -1, "The port that the service should serve on. Copied from the resource being exposed, if unspecified")
 	cmd.MarkFlagRequired("port")
-	cmd.Flags().String("type", "", "Type for this service: ClusterIP, NodePort, or LoadBalancer. Default is 'ClusterIP' unless --create-external-load-balancer is specified.")
+	cmd.Flags().String("type", "", "Type for this service: ClusterIP, NodePort, or LoadBalancer. Default is 'ClusterIP'.")
+	// TODO: remove create-external-load-balancer in code
```
Comment (thockin) [inline pkg/kubectl/cmd/expose.go]: You should put a date here - on or after Aug 25, 2016.

**Comment by thockin** [comment]
Just a comment nit.  Thanks

File: pkg/kubectl/cmd/expose.go (hunk 1 above)
Comment (JanetKuo) [inline pkg/kubectl/cmd/expose.go]: Done

**Comment by thockin** [comment]
LGTM but CI failed because of doc generation (which I swear was in the first cut of this PR?)

**Comment by JanetKuo** [comment]
@thockin hmm something went wrong while rebasing... it's fixed now.

**Comment by JanetKuo** [comment]
@k8s-bot test this please

**Comment by JanetKuo** [comment]
@k8s-bot test this please

**Comment by JanetKuo** [comment]
@thockin PTAL

**Comment by eparis** [comment]
stealing, @thockin has better things to do.

**Pull request merged**